cmd/state-offline-installer/install.go

```diff
@@ -199,15 +208,15 @@ func (r *runner) Run(params *Params) error {
 
 		// create batch script
 		batch := fmt.Sprintf(
-			"@echo off\ncopy %s\\%s %%TEMP%%\\%s\n%%TEMP%%\\%s %s & del %%TEMP%%\\%s >nul 2>&1\n",
+			"@echo off\ncopy %s\\%s %%TEMP%%\\%s >nul 2>&1\n%%TEMP%%\\%s %s & del %%TEMP%%\\%s >nul 2>&1 & echo You can safely ignore the following error message:\n",
 			uninstallDir,
 			uninstallerDestName,
 			uninstallerDestName,
 			uninstallerDestName,
-			params.path,
+			installDir,
 			uninstallerDestName,
 		)
-		err := os.WriteFile(filepath.Join(params.path, "uninstall.bat"), []byte(batch), 0755)
+		err = os.WriteFile(filepath.Join(installDir, "uninstall.bat"), []byte(batch), 0755)
```

Comment:
Is this fixing something unrelated to this PR? I feel uncertain reviewing this without knowing the context.

Reply:
It's tangentially related in that the uninstaller couldn't ever send a success event without this mechanism, as it would always fail trying to delete itself on windows.  Windows doesn't allow you to delete currently running executables as it pages them directly from their location on disk rather than copying them either to memory or a page file like UNIX systems do.